code/game/objects/items/tanks/tanks.dm

```diff
@@ -48,33 +48,53 @@
 /obj/item/tank/ui_action_click(mob/user)
 	toggle_internals(user)
 
-/obj/item/tank/proc/toggle_internals(mob/user)
-	var/mob/living/carbon/human/H = user
-	if(!istype(H))
+/obj/item/tank/proc/open_internals(mob/living/carbon/human/human_target)
+	if(human_target.internal)
+		to_chat(human_target, span_notice("You switch your internals to [src]."))
+	else
+		to_chat(human_target, span_notice("You open [src] valve."))
+	human_target.open_internals(src)
+/obj/item/tank/proc/close_internals(mob/living/carbon/human/human_target)
+	to_chat(human_target, span_notice("You close [src] valve."))
+	human_target.close_internals()
+
+/obj/item/tank/proc/toggle_internals(mob/mob_target)
```

Comment:
DMDoc what these procs do, you can read more about that here: (https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md)

Reply:
Fixed by commit c7d5cf61ea922486cdc0586fa494b3c339926c96